MotionMark/resources/runner/motionmark.js

```diff
@@ -321,14 +322,14 @@ ResultsTable = Utilities.createClass(
         });
 
         this.clear();
-    }, {
+    }
 
-    clear: function()
+    clear()
     {
         this.element.textContent = "";
-    },
+    }
 
-    _addHeader: function()
+    _addHeader()
```

Comment:
Should we use the private syntax for JS classes: `#addHeader()`?

Reply:
We should, as long as browser support is good enough.